src/modules/webpack-publish.ts

```diff
@@ -57,20 +70,21 @@ const WebpackPublish = {
 
         // Build frontend
         Echo.message(`Building frontend (via ${this.cmd()})...`);
+        const { cmd, args } = this.cmd();
+        const { status: buildStatus } = child_process.spawnSync(cmd, args, {
+            stdio: "inherit",
```

Comment:
I admitedly don't know much about `spawnSync` and and more specifically the `stdio` parameter, however I would suggest making `inherit` a constant and defining it's hard coding somewhere to prevent duplicate hard coded strings assuming it's a string that would be used from multiple reference points.

Reply:
`inherit` is strongly typed from the `child_process` module, so it's not exactly hardcoded - 
 `type StdioOptions = "pipe" | "ignore" | "inherit" | Array<("pipe" | "ipc" | "ignore" | "inherit" | Stream | number | null | undefined)>;`
 
 However, the configuration options object we pass to `spawnSync` is pretty much identical across the board - it's one of the main motivations for #160, aside from also refactoring out the mix of `shell.exec`s we have lingering around. I'll leave it in for now since there's a documented issue to go back and abstract it further.